dbscripts/xml/upgrade.xml

```diff
@@ -124,6 +124,7 @@
 
 	<upgrade minversion="3.4.0.0" maxversion="3.4.9.9">
 		<migration class="APP\migration\upgrade\v3_4_0\I9813_QuickSubmitSubmissionProgressType"/>
+		<migration class="APP\migration\upgrade\v3_4_0\I11125_UpdateEmailTemplateVariables"/>
```

Comment:
I think it makes sense to fix the problem at the origin (I didn't inspect the issue very well, but according to your comment it's here: `classes/migration/upgrade/v3_4_0/I7264_UpdateEmailTemplates.php`). I mean, if we shouldn't rename the variable there, just remove it.

Then we can apply this migration only to the affected installations, so you should create an `<upgrade>` entry to match the affected range (minversion = when the defect was created, maxversion = current release).

Reply:
Agreed, we shouldn't be referencing 3.4.0 migrations in the 3.4.0 to 3.5.0 upgrade.